### DISCUSSION TITLE
Remaining Dollar amount in Category list.

### DISCUSSION THREAD

**Discussion by bpaulien** [actualbudget/releases#62] [Feature ideas]
I think it would be nice to show the amount remaining (Balance from the budget) in the name of the category, when you are entering a new transaction.  Sometimes I don't remember how much is in a given category, or don't feel like going to look it up after I've entered a transaction.  Let's say I have budgeted $300 for Groceries this month.  If I have $200 left, it would be nice to see "Groceries ($200)" (format can be different, just the fact that it displays it somehow) as the category name, so now when I enter a transaction for $90 groceries, I know, OK, there's about $110 left.  I don't have to go searching to see if I went over, or how much I have left.

**Comment by bdoherty** [comment]
The title of this issue doesn't seem to match the description. Should the title be: **Add the ability to sort Schedules (ASC and DESC) by clicking column header**

**Reply by bpaulien** [reply]
No, that's really weird.  I already put in one for that.  This one seems to have the exact same text in it as the other.  Somehow the text didn't change when I submitted the second one.  Let me see if I can remember what this was, and edit the content of this one.

**Comment by Teprifer** [comment]
Agreed, I like the little popup YNAB mobile app has where it briefly shows a floaty bit at the bottom with the category name and then the new balance. 
Well actually it pops up for under a second with the old balance, then again for a similar amount of time with the new balance.
